src/common.js

```diff
@@ -113,7 +113,7 @@ export function resolveUrl (relUrl, parentUrl) {
 function resolveAndComposePackages (packages, outPackages, baseUrl, parentMap) {
   for (let p in packages) {
     const resolvedLhs = resolveIfNotPlainOrUrl(p, baseUrl) || p;
-    if (outPackages[resolvedLhs] && (outPackages[resolvedLhs] !== packages[resolvedLhs])) {
+    if ((!shimMode || !override) && outPackages[resolvedLhs] && (outPackages[resolvedLhs] !== packages[resolvedLhs])) {
```

Comment:
Rather than moving the "noop" to a separate file, what about creating a separate `resolve.js` module?

Reply:
Good call, let me give it a shot!

Reply:
Addressed here: https://github.com/guybedford/es-module-shims/pull/257/commits/8140a862d64fa661533b6f4ea2367044bef8724d